.eslintrc

```diff
@@ -61,6 +61,14 @@
         "@typescript-eslint/no-non-null-assertion": "off",
         "no-lone-blocks": "off"
       }
+    }, {
+      "files": [
+        "**/jellyfish-api-core/__tests__/**/*.test.ts",
```

Comment:
Appy this new rule to` jellyfish-api-core` first

Reply:
Once I have applied the changes to all packages, I will remove the all the **package-specific** settings in **overrides** section and add a global setting in **rules** section